example/feature_demo/demo_service.pb.gorm.go

```diff
@@ -103,52 +103,82 @@ func DefaultCreateIntPoint(ctx context.Context, in *IntPoint, db *gorm1.DB) (*In
 	if err != nil {
 		return nil, err
 	}
+	if hook, ok := interface{}(&ormObj).(IntPointORMWithBeforeCreate); ok {
+		db, err = hook.BeforeCreate(ctx, db)
+		if err != nil {
```

Comment:
I see a lot of places using this form instead of the more condensed `if {db,} err = ...; err != nil {` form. Is there a reason behind that, or is it just coincidental?

Reply:
I don't know.. Perhaps I like this way because it's quite easy to copy-paste around.. But honestly I don't care much about this..